src/components/course/data/hooks.js

```diff
@@ -200,6 +243,7 @@ export function useCoursePriceForUserSubsidy(activeCourseRun, userSubsidy) {
   return [coursePrice, currency];
 }
 
+// TODO Remove location here and replace with just enrollmentFailedParams...
```

Comment:
Is this TODO something you're planning for this PR? Don't mind either way, just curious!

Reply:
done in new pr!